### ISSUE TITLE
About the queried_frame of tracks

### ISSUE THREAD

**Issue by Ishida-Mitsunari** [facebookresearch/co-tracker#156]
Hello, the cotracker3_online uses these code before `forward_window()` and multiplies the `attention_mask` and `track_feat_support_pyramid`:
```
attention_mask = (queried_frames < ind + S).reshape(B, 1, N)
```
Therefore it seems that the `queried_frame` of each track in the current window is either ≤  `ind` or greater than `ind + S`(its `track_feat_support` multiplies 0 or not), without the situation in which  `ind < queried_frame < ind + S`. Isn't it?

Moreover, I notice that the `attention_mask` parameter of `forward_window()` isn't used and nothing is passed to the `mask` parameter of `forward()` in `EfficientUpdateFormer`.
